### DISCUSSION TITLE
Remember Location of External Drive? (Feature Request)

### DISCUSSION THREAD

**Discussion by 630216-29-2400** [RetroDriven/Pocket_Updater#5] [Ideas]
Not an issue as it is in any way but if at all possible could a future revision have the option to bring the previously used Update Location up as default the next time you open the GUI rather than having to navigate to a location again?

**Comment by RetroDriven** [comment]
I assume this is for when you select the Removable Storage Location? And have multiple Removable Drives connected.

**Comment by 630216-29-2400** [comment]
That’s right, yes.
Just simplifes things a little for the end user. It may not justify the work needed on your end though.

**Comment by RetroDriven** [comment]
I just enabled the Discussions option and merged it over here.
